Formula/rclone.rb

```diff
@@ -0,0 +1,17 @@
+class Rclone < Formula
+  desc "Sync files to and from Google Drive, Amazon Cloud Drive, etc."
+  homepage "http://rclone.org"
+  url "https://github.com/ncw/rclone/releases/download/v1.29/rclone-v1.29-osx-amd64.zip"
```

Comment:
We prefer to build this from source instead of installing a precompiled binary. See [this part of our Acceptable Formulae](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#we-dont-like-binary-formulae) document.